src/plugins/advanced_settings/public/management_app/advanced_settings.tsx

```diff
@@ -257,10 +262,15 @@ export class AdvancedSettingsComponent extends Component<
 }
 
 export const AdvancedSettings = (props: AdvancedSettingsProps) => {
```

Comment:
nit: it's not a question to your changes but looks like this wrapper for **AdvancedSettingsComponent** was added just to use **useParams** hook. Honestly I think this wrapper should be removed. 

we can probably read **queryText** from `props.history.location.pathname` (see my previous comment)

Reply:
see 336f39e